dataland-batch-manager/src/test/kotlin/org/dataland/batchmanager/service/GleifGoldenCopyIngestorTest.kt

```diff
@@ -0,0 +1,58 @@
+package org.dataland.batchmanager.service
+
+import com.fasterxml.jackson.databind.MappingIterator
+import org.dataland.datalandbackend.openApiClient.api.ActuatorApi
+import org.dataland.datalandbatchmanager.service.CompanyUpload
+import org.dataland.datalandbatchmanager.service.GleifApiAccessor
+import org.dataland.datalandbatchmanager.service.GleifCsvParser
+import org.dataland.datalandbatchmanager.service.GleifGoldenCopyIngestor
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito.any
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.reset
+import org.mockito.Mockito.`when`
+import java.io.BufferedReader
+import java.io.File
+
+class GleifGoldenCopyIngestorTest {
+    private val mockGleifApiAccessor = mock(GleifApiAccessor::class.java)
+    private val mockGleifCsvParser = mock(GleifCsvParser::class.java)
+    private val mockCompanyUpload = mock(CompanyUpload::class.java)
+    private val mockActuatorApi = mock(ActuatorApi::class.java)
+    private lateinit var companyIngestor: GleifGoldenCopyIngestor
+
+    @BeforeEach
+    fun setupTest() {
+        reset(mockGleifApiAccessor)
+        reset(mockGleifCsvParser)
+        reset(mockCompanyUpload)
+        reset(mockActuatorApi)
+    }
+
+    @Test
+    fun `test failing ingestion`() {
```

Comment:
Description is a little misleading.
This test checks that "nothing" (besides a log) happens if `processFullGoldenCopyFileIfEnabled()` is executed with the `allCompaniesForceIngest = false` and a missing flagFile
In my opinion it should also be checked that `processFile` does not run, showcasing that the `else`-block of `processFullGoldenCopyFileIfEnabled()` has actually been executed.

Reply:
TEst wird so wie er jetzt ist immer grün sein, egal ob der `if` oder `else` Block von `processFullGoldenCopyFileIfEnabled()` ausgeführt wird.

Reply:
Fixed that